sim/dex.ts

```diff
@@ -953,6 +953,7 @@ export class ModdedDex {
 		switch (rule.charAt(0)) {
 		case '-':
 		case '+':
+			rule = rule.trim();
```

Comment:
```suggestion
		rule = rule.trim();
		switch (rule.charAt(0)) {
		case '-':
		case '+':
```

(This is necessary if the rule starts with a space.)